packages/gitgraph-core/package.json

```diff
@@ -71,7 +71,7 @@
   },
   "jest": {
     "transform": {
-      ".(ts)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+      ".(ts)": "<rootDir>/../../node_modules/ts-jest/preprocessor.js"
```

Comment:
`ts-jest` is now shared, so on `/node_modules` instead of `/packages/gitgraph-core/node_modules` (yarn workspaces)